go.mod

```diff
@@ -95,6 +95,16 @@ require (
 	upper.io/db.v3 v3.7.1+incompatible
 )
 
+require (
```

Comment:
Why is a separate require required here?

Reply:
merged into one part.